charmtools/build/tactics.py

```diff
@@ -491,6 +501,14 @@ def combine(self, existing):
 
     def apply_edits(self):
         super(MetadataYAML, self).apply_edits()
+        # Remove the merged maintainers from the self.data
+        self.data.pop('maintainer', None)
+        self.data.pop('maintainers', [])
+        # Set the maintainer and maintainers only from this layer.
+        if self.maintainer:
+            self.data['maintainer'] = self.maintainer
+        if self.maintainers:
```

Comment:
@marcoceppi I considered doing this and discussed it with @johnsca. I came to the conclusion that we should not rename fields, if the author used the "maintainer" key we should keep the same key in the built artifact. Also I could not find anything that said the "maintainer" key was deprecated, to my knowledge it is still a valid key. A new bug would clarify this and I would be happy to change the behavior.